.travis.yml

```diff
@@ -17,7 +17,9 @@ before_script:
     - docker-compose up -d
 
 script:
+    - bash -xc "shellcheck .docker/*.sh"
```

Comment:
`shellcheck` is not a shell script so the `-x` should make no difference?

Reply:
Copied from https://github.com/libero/scripts/blob/bf3363db288d1940ce705e7e4e327c92c7cb5340/.travis.yml#L4

Reply:
IIRC it's to [list the files that are found by `*.sh`](https://github.com/libero/scripts/pull/1/commits/23ec3d31b5b69a10a32eba91f026067b8942ac84).